Makefile

```diff
@@ -99,6 +99,7 @@ ifeq (,$(wildcard aws-nitro-enclaves-cli))
 	git clone https://github.com/aws/aws-nitro-enclaves-cli.git
 endif
 	cd "aws-nitro-enclaves-cli" && git checkout $(AWS_NITRO_CLI_REVISION)
+	perl -i -pe 's/readlink -f/realpath/' aws-nitro-enclaves-cli/Makefile
```

Comment:
A comment to explain why this is necessary would be helpful. I think this worked for me without the `perl -i` patch.

Reply:
This is for building on Mac